packages/phpstan-rules/src/Rules/Explicit/NoMixedMethodCallerRule.php

```diff
@@ -41,7 +42,10 @@ public function process(Node $node, Scope $scope): array
             return [];
         }
 
-        return [self::ERROR_MESSAGE];
+        /** @var string $name */
+        $name = $node->name instanceof Variable ? $node->name->name : $node->name;
```

Comment:
Here you need to check for the instance of `$node->name`:

```php
if ($node->name instanceof Identifier) {
    $methodName = $node->name->toString();
} else {
    // unknown name
    $methodName = '';
}
```

Reply:
So in the end the method name does not matter much, as we target the caller (left part). Sorry for the confusion.